src/software/jetson_nano/services/motor.h

```diff
@@ -107,9 +108,11 @@ class MotorService
      * @param motor The motor to configure (the same value as the chip select)
      */
     void configurePWM(uint8_t motor);
-    void configurePI(uint8_t motor);
+    void configureDribblerPI(uint8_t motor);
```

Comment:
does the input `uint8_t` correspond to the `SPI_CHIP_SELECTS` found in the top of `motor.cpp`?

Reply:
yup thats correct